vagrant/boxes.d/99-local.yaml.example

```
@@ -1,7 +1,25 @@
 ---
-centos8-katello-devel:
+centos9-katello-devel:
   primary: true
-  box: centos8-stream
+  box: centos9-stream
+  ansible:
+    playbook: 'playbooks/katello_devel.yml'
+    group: 'devel'
+    variables:
+      ssh_forward_agent: true
+      foreman_devel_github_push_ssh: True
+      katello_devel_github_username: <REPLACE ME>
+      foreman_repositories_environment: staging
+      katello_repositories_environment: staging
+      foreman_repositories_version: nightly
+      katello_repositories_version: nightly
+      foreman_installer_options:
+        - "--foreman-proxy-content-enable-ostree=true"
+#        - "--katello-devel-modulestream-nodejs=18"
```

I think commenting in this style is easier to read. Is that just me?
```suggestion
        #- "--katello-devel-modulestream-nodejs=18"
```

---

I feel like your way is nicer to look at, but having it on the right side is quicker to delete in a terminal editor (only have to go down instead of down and right). 

The method I chose was pretty much the result of a coin flip :)

---

VSCode has an easy keyboard shortcut to uncomment the current line, just saying..

---

I never thought about it because in vim `^` navigates to the first non-whitespace character of a line so `^x` deletes the comment.

---

That's a good tip. I moved the comment.